setup.py

```diff
@@ -16,9 +16,9 @@
     long_description_content_type="text/markdown",
     long_description=pathlib.Path("README.md").read_text("utf-8"),
     install_requires=[
-        "boto3>=1.34.11,<1.35.0",
-        "multidict>=6.0.4,<6.1.0",
-        "python-jose>=3.3.0,<3.4.0",
+        "boto3>=1.38.0,<1.39.0",
+        "multidict>=6.4.0,<6.5.0",
+        "python-jose>=3.4.0,<3.5.0",
```

Comment:
I also wanted to make this change, just to make our life easier, but what about https://github.com/pdyba/lambdalizator/pull/103? 🤔